packages/cli/src/tools/config/readConfigFromDisk.js

```diff
@@ -123,11 +125,5 @@ export function readLegacyDependencyConfigFromDisk(
     )} is using deprecated "rnpm" config that will stop working from next release. Consider upgrading to the new config format.`,
   );
 
-  const result = Joi.validate(transformedConfig, schema.dependencyConfig);
```

Comment:
why removing validation, is to done somewhere else already?

Reply:
Yes, this function is now not exported to the outside world. The return value of this function will be validated in the `readConfigFromDisk` function.

Reply:
[Here](https://github.com/react-native-community/cli/pull/395/files#diff-f06b7f8213679acb5163a64a52db952bR62), to be precise